packages/hoprd/releases.json

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    "id": "master-goerli",
+    "git_ref": "refs/heads/master",
+    "environment_id": "master-goerli"
+  },
+  {
+    "id": "wildhorn",
+    "version_major": 1,
+    "version_minor": 76,
+    "git_ref": "refs/heads/release/wildhorn",
+    "environment_id": "wildhorn"
+  },
+  {
+    "id": "constantine",
+    "version_major": 1,
+    "version_minor": 75,
+    "git_ref": "refs/heads/release/constantine",
+    "deprecated": true
+  },
+  {
+    "id": "moscow",
```

Comment:
I'd rather not have these old releases in here since they are not supported in any capacity by the build system as we will use it in any form.